llvm/lib/Target/RISCV/RISCVInstrInfoZvk.td

```diff
@@ -150,6 +150,14 @@ let Predicates = [HasStdExtZvkg], RVVConstraint = NoConstraint in {
                  SchedBinaryMC<"WriteVGMULV", "ReadVGMULV", "ReadVGMULV">;
 } // Predicates = [HasStdExtZvkg]
 
+let Predicates = [HasStdExtZvkgs], RVVConstraint = NoConstraint in {
```

Comment:
```suggestion
let Predicates = [HasStdExtZvkgs], RVVConstraint = VS2Constraint in {
```
Reserved Encodings
+ SEW is any value other than 32
+ **the vd register group overlaps the vs2 scalar element group**

Reply:
Oops, I think we should set this for `vghsh.vv` and `vgmul.vv` as well. cc @4vtomat

Reply:
I guess the spec didn't state that for `.vv` version?

Reply:
Why? I was thinking it should be the same case?

Reply:
I made a comment in https://github.com/riscv/riscv-isa-manual/pull/1306. We may not need this constraint to keep the consistency between `.vv` and `.vs`.

Reply:
We already have other instructions that aren't consistent between `.vv` and `.vs`. For example, vaesdf.vv and vaesdf.vs.